src/test/end-to-end/codecept.conf.js

```diff
@@ -7,6 +7,7 @@ exports.config = {
         'Puppeteer': {
             'waitForTimeout': 60000,
             'getPageTimeout': 60000,
+            'waitForAction': 500,
```

Comment:
what do you need this for? (waitForAction)
doe not look like its being used anywhere